platform/javascript/detect.py

```diff
@@ -20,6 +22,14 @@ def get_opts():
     from SCons.Variables import BoolVariable
 
     return [
+        ("initial_memory", "Initial WASM memory (in MiB)", 16),
+        BoolVariable("use_assertions", "Use emscripten runtime assertions", False),
+        BoolVariable("use_thinlto", "Use ThinLTO", False),
+        BoolVariable("use_ubsan", "Use LLVM/GCC compiler undefined behavior sanitizer (UBSAN)", False),
+        BoolVariable("use_asan", "Use LLVM/GCC compiler address sanitizer (ASAN)", False),
+        BoolVariable("use_lsan", "Use LLVM/GCC compiler leak sanitizer (LSAN)", False),
```

Comment:
Nitpick but in this case it can't be GCC I guess, it's Emscripten/LLVM.

Reply:
True, I copied it from other platforms.
And since it's like that in `master` now I would prefer leaving it like that for now (so I can keep them more in sync).
I'll try to remember to change it next time I PR some JS stuff that needs backporting.